runtime-testsuite/test/org/antlr/v4/test/runtime/javascript/browser/BaseBrowserTest.java

```diff
@@ -656,7 +656,6 @@ protected void writeParserTestFile(String parserName,
 			"				};\r\n" +
 			"\r\n" +
 			"				TreeShapeListener = function() {\r\n" +
-			"					antlr4.tree.ParseTreeListener.call(this);\r\n" +
```

Comment:
how about converting this test class to ES6?

Reply:
I thought the main point of the browser tests was that they should work everywhere ? if I convert this to es6 classes then we wouldnt be able to do that right ?

Reply:
The browser tests are not run as part of the CI, but I suspect they will fail...

Reply:
I tried running them and indeed they fail. Not only because of this refactoring but it seems that the previous ones also added some errors 😞 

The browser tests are still using `lib/require` even though we removed it. I rolled that back locally but still ran into errors due to the missing module `fs`. Since the whole library is expected to be webpacked.

I see two options right now:
- modify the browser test suite to use webpack
- rollback some of the changes for lib/require and conditional `fs` module import

Let me know what you think.

Reply:
Preferred option is "modify the browser test suite to use webpack"
I did that in another project (run webpack from Java) and it's quite a piece of work...

Reply:
> (run webpack from Java) and it's quite a piece of work...

The "piece of work" was meant ironically ? My first view of the setup for those tests is that it is definitely not going to be easy :/

I was already inclining towards a rollback and leave this test suite to the end 🙈

Reply:
It's not my IP unfortunately 

Another approach could be to pre-webpack the text class making the parent class external?